apps/trading/components/rewards-container/rewards-container.tsx

```diff
@@ -54,6 +61,30 @@ export const RewardsContainer = () => {
 
   const { data: epochData } = useRewardsEpochQuery();
 
+  const { data: partyActivityStreak } = useActivityStreakQuery({
```

Comment:
Can remove these extra queryes if you just add them to the rewards page query

Reply:
yes, removed